src/__tests__/Home.test.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import NavBar from '../components/NavBar';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(
+      <div className="home">
+        <BrowserRouter>
+          <NavBar />
+        </BrowserRouter>
+        <h1>Math Magicians</h1>
```

Comment:
- [ ] Kindly enable eslint in this module. It's important to run linting checks on all modules and components.